identity-documents/README.md

```diff
@@ -0,0 +1,185 @@
+# Identity Documents as Receipts
+
+The simplest form of identity document is a signed public key.
+
+- unknown.
```

Comment:
I think this is supposed to be a quote? Current formatting makes it look like there's just a bullet point that says "unknown" after the first sentence, I realized it might be a quote so figured it might need a quote block.

```suggestion
> The simplest form of identity document is a signed public key.
>
> - *unknown.*
```

Reply:
Maybe:

```suggestion
"The simplest form of identity document is a signed public key."

-- unknown
```